bundles/org.openhab.binding.omnikinverter/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -9,20 +9,93 @@
 		<description>Thing for OmnikInverter Binding</description>
 
 		<channels>
-			<channel id="power" typeId="power"/>
-			<channel id="powerAC1" typeId="powerAC1"/>
-			<channel id="powerAC2" typeId="powerAC2"/>
-			<channel id="powerAC3" typeId="powerAC3"/>
-			<channel id="currentPV1" typeId="currentPV1"/>
-			<channel id="currentPV2" typeId="currentPV2"/>
-			<channel id="currentPV3" typeId="currentPV3"/>
-			<channel id="voltagePV1" typeId="voltagePV1"/>
-			<channel id="voltagePV2" typeId="voltagePV2"/>
-			<channel id="voltagePV3" typeId="voltagePV3"/>
-			<channel id="energyToday" typeId="energyToday"/>
-			<channel id="energyTotal" typeId="energyTotal"/>
+			<channel id="power" typeId="system.electric-power">
+				<label>Instantaneous Power</label>
+				<description>The instantaneous power generation for output 1</description>
+			</channel>
+			<channel id="powerAC1" typeId="system.electric-power">
```

Comment:
This is actually a breaking change until openhab/openhab-core#2590 has been resolved, unfortunately. Even though the channel type is fully compatible, I believe this could be a problem for existing things configured in the UI.

You would either need to revert the channel type changes for **existing** channels or add a note here:
https://github.com/openhab/openhab-distro/blob/84f0e7d06e23befbb7a9bebe36783fc5f1665f18/distributions/openhab/src/main/resources/bin/update.lst#L95-L99

IMHO your refactoring/simplification makes sense, so either solution is fine by me.

Reply:
I mean that changing the underlying channel type for an existing channel is a breaking change (when the previous channel type id no longer exists). I'm fine with it, but it needs to be mentioned in the openhab-distro notes that users may have to re-create their things. You should be able to reproduce it if you go back to the previous version before your changes and create an item for one of these old channels - in the UI, so it will be managed. Then upgrade to this version.

Reply:
Unresolving this for visibility until PR towards openhab-distro has been created. @leansoft, please remember about this.